css/src/components/table.scss

```diff
@@ -0,0 +1,67 @@
+$table-cell-padding-xxs: $spacer-4 !default;
```

Comment:
`$table-cell-vertical-padding`?
`$table-cell-horizontal-padding`?

-xss and -xs make it feel like there are other options, but that isn't really the case

Reply:
Updated